fe/fe-core/src/main/java/com/starrocks/catalog/HudiTable.java

```diff
@@ -39,14 +40,13 @@
 import java.io.DataInput;
 import java.io.DataOutput;
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
 
-public class HudiTable extends Table {
+public class HudiTable extends Table implements HMSTable {
```

Comment:
add some annotation for this class, since hudi may support non-hms based catalog.we should explain it here

Reply:
done